--- libdlna-0.2.3/configure	2007-11-26 21:47:43.000000000 +0100
+++ libdlna-0.2.3/configure.new	2007-12-14 22:40:43.000000000 +0100
@@ -270,7 +270,7 @@
 cat <<EOF >$name.pc
 prefix=$PREFIX
 exec_prefix=\${prefix}
-libdir=\${exec_prefix}/lib
+libdir=${libdir}
 includedir=\${prefix}/include
 
 Name: $name
@@ -597,7 +597,7 @@
     add_cflags  "-O5"
     add_ldflags "-O5"
   else
-    add_cflags "-O3"
+    add_cflags ""
   fi
 fi
 
@@ -697,7 +697,7 @@
 #################################################
 #   make pkg-config files
 #################################################
-pkgconfig_generate libdlna "DLNA (Digital Living Network Alliance) library" "$VERSION" "-ldlna $extralibs" "libavformat libavcodec"
+pkgconfig_generate libdlna "DLNA (Digital Living Network Alliance) library" "$VERSION" "-ldlna" "libavformat libavcodec"
 
 clean
 exit 0
